notebooks/PhasePlane.ipynb

```diff
@@ -50,10 +50,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "id": "acd78ac4-2703-444f-bb6c-1d7080b6552d",
-   "metadata": {},
-   "outputs": [],
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "09-12-2022 12:30:41 - DEBUG - tvbwidgets - Package is not fully installed\n",
+      "C:\\Users\\teodora.misan\\Documents\\tvb-widgets\n",
+      "09-12-2022 12:30:41 - DEBUG - tvbwidgets - Version read from the internal package.json file\n",
+      "09-12-2022 12:30:41 - INFO - tvbwidgets - Version: 1.0\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\teodora.misan\\Anaconda3\\envs\\tvbwidgets\\lib\\site-packages\\pkg_resources\\__init__.py:123: PkgResourcesDeprecationWarning: -PKG-VERSION is an invalid version and will not be supported in a future release\n",
+      "  warnings.warn(\n",
+      "C:\\Users\\teodora.misan\\Anaconda3\\envs\\tvbwidgets\\lib\\site-packages\\tvb\\datatypes\\surfaces.py:64: UserWarning: Geodesic distance module is unavailable; some functionality for surfaces will be unavailable.\n",
```

Comment:
We try to avoid having user paths on GIT, thus, before we save a notebook, we try to execute clear outputs. 
I think that if you do that, the noteboook, will not have changes almost at all.